### PR TITLE
 Enhancement to make application name configurable for 2FA

### PR DESCRIPTION
**Pull request by neeraj890kumar**
Resolves: #1900

### CODE REVIEW

File: confd/templates/gui/application.properties.tmpl [hunk 1]
```diff
@@ -54,3 +54,5 @@ spring.mvc.view.prefix= /views/
 spring.mvc.view.suffix= .jsp
 
 status.cron.time=43200000
+
+twofa.application.name=Open Kilda
```
Comment (ashishaggarwalCS) [inline confd/templates/gui/application.properties.tmpl]: Change this to application name

File: services/src/openkilda-gui/src/main/java/org/openkilda/controller/LoginController.java [hunk 2]
```diff
@@ -70,6 +71,9 @@
 
     @Autowired
     private PermissionRepository permissionRepository;
+    
+    @Value("${twofa.application.name}")
+    private String twofaApplicationName;
```
Comment (ashishaggarwalCS) [inline services/src/openkilda-gui/src/main/java/org/openkilda/controller/LoginController.java]: Remove twofa the property name should be application name only

File: confd/templates/gui/application.properties.tmpl (hunk 1 above)
Comment (hiteshgupta1) [reply confd/templates/gui/application.properties.tmpl]: changes done

File: services/src/openkilda-gui/src/main/java/org/openkilda/controller/LoginController.java (hunk 2 above)
Comment (hiteshgupta1) [reply services/src/openkilda-gui/src/main/java/org/openkilda/controller/LoginController.java]: changes done

**Pull request merged**